src/config/configure.in

```diff
@@ -844,11 +844,19 @@ AS_HELP_STRING([--with-fmangle=FMANGLE],
                [FMANGLE contains a string indicating the type of name mangling
                 to use when calling hypre from Fortran.  It can be set to:
                 "no-underscores", "one-underscore", "two-underscores",
-                "caps-no-underscores, and "one-before-after".]),
+                "caps-no-underscores", and "one-before-after".]),
 [hypre_fmangle=0; AC_HYPRE_SET_FMANGLE],
 [hypre_fmangle=0]
 )
-AC_DEFINE_UNQUOTED(HYPRE_FMANGLE, [$hypre_fmangle], [Fortran mangling])
+AC_DEFINE_UNQUOTED(HYPRE_FMANGLE, 
+		   [$hypre_fmangle],
+[Define as follows to set the Fortran name mangling scheme:
+ 0 = unspecified
```

Comment:
@pbauman Maybe add a semicolon at the end of each of these lines to help with the formatting in the `.h` file?  Looks great otherwise.